docs/master/api-reference/scalars.md

```diff
@@ -0,0 +1,55 @@
+# Scalars
+Scalars will parse the input and serialize the output of attributes and nodes of said type.
+
+**Schema example:**
+```graphql
+scalar DateTime @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\DateTime")
+
+type Appointment {
+    title: String!
+    date: DateTime!
+    created_at: DateTime!
+    updated_at: DateTime!
+}
+
+type Query {
+    appointments: [Appointment] @paginate(type: "connection")
+}
+
+type Mutation {
+    addAppointment(text: String!, date: DateTime!): Appointment @create
+}
+
+```
+
+## Date
+**Parse:**
+
+Expects an ISO date string, eg.: `2019-01-15`. Parses to a Carbon-object.
+
+**Serialize:**
+
+Convert a date or Carbon-object to an ISO 8601 date string, eg. `2019-01-15`.
+
+## DateTime
+
+**Parse:**
+
+Expects an ISO 8601 datetime string, eg.: `2019-01-15T23:15:33`. Parses to a `Carbon`-object.
```

Comment:
This is actually not correct, the formatter it's trying to parse is `Y-m-d H:i:s`, and ISO 8601 is defined as `Y-m-d\TH:i:sO`.